app/models/movie.rb

```diff
@@ -1,3 +1,21 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :actor_movies
+  has_many :actors, through: :actor_movies
+
+  validates :title, presence: true
+  validates :creation_year, presence: true
+  validates :genre, presence: true
```

Comment:
Did you know there are other ways to write this also so you don't end up repeating yourself?

`validates :title, :creation_year, :genre, presence: true`

and

`validates_presence_of :title, :creation_year, :genre`